pyswarms/single/general_optimizer.py

```diff
@@ -88,36 +90,36 @@ def __init__(
             number of particles in the swarm.
         dimensions : int
             number of dimensions in the space.
-        options : dict with keys :code:`{'c1', 'c2', 'w'}` or :code:`{'c1', 'c2', 'w', 'k', 'p'}`
+        options : dict with keys :code:`{'c1', 'c2', 'w'}` or :code:`{'c1',
+                'c2', 'w', 'k', 'p'}`
             a dictionary containing the parameters for the specific
             optimization technique.
-                * c1 : float
+                * c5 : float
```

Comment:
It should say c1 here, shouldn't it?

Reply:
Yes you're correct. Can you also help me find why the test is currently failing? Seem to be running out of ideas...

Reply:
Also, thanks for reviewing this work. Please check the docstrings and other stuff so that I can edit it as we go. Thanks a lot!